source/photo.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Пинк</title>
+  </head>
+  <body>
+    <header class="page-header page-header--photo">
+      <div class="page-header__wrapper">
+        <a class="page-header__logo" href="#">
+          <img class="page-header__logo-image" src="" width="74" height="23" alt="На главную страницу сайта приложения Пинк.">
+        </a>
+        <nav class="page-header__navigation main-navigation">
+          <button class="main-navigation__toggle" type="button">
+            <span class="visually-hidden">Открыть меню.</span>
+          </button>
+          <ul class="main-navigation__list site-list">
+            <li class="site-list__item">
+              <a>Главная</a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">Фотографии</a>
+            </li>
+            <li class="site-list__item">
+              <a href="#">Конкурс</a>
+            </li>
+            <li class="site-list__item">
+              <a href="https://htmlacademy.ru/intensive/adaptive" target="_blank">HTML Academy</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <h1 class="page-header__header page-header__header--photo">Фотографии пользователей</h1>
```

Comment:
```suggestion
      <h1 class="page-header__header page-header__header--form">Фотографии пользователей</h1>
```

тут такой же мод как на форме, тк они одинаковые. Может ты захочешь ему другое название дать. А вообще, тк этот мод используется на 2-х страницах уже, а индекс один, лучше это сделать дефолтом, а на индексе добавить мод.

Всегда удобнее то, что используется чаще делать дефолтным стилем, а что-то исключительное - с помощью модификатора